openpype/hosts/max/plugins/publish/collect_render.py

```diff
@@ -97,8 +97,8 @@ def process(self, instance):
             "renderer": renderer,
             "source": filepath,
             "plugin": "3dsmax",
-            "frameStart": int(rt.rendStart),
-            "frameEnd": int(rt.rendEnd),
+            "frameStart": instance.data.get("frameStartHandle"),
+            "frameEnd": instance.data.get("frameEndHandle"),
```

Comment:
If this wants the data from `CollectFrameRange` this should be `CollectorOrder + 0.02`
```suggestion
            "frameStart": instance.data["frameStartHandle"],
            "frameEnd": instance.data["frameEndHandle"]
```